src/Input/inputStyles.ts

```diff
@@ -0,0 +1,28 @@
+import styled from '@emotion/styled';
+
+export const StyledLabel = styled.label(({ theme }) => ({
+    color: theme?.colors?.cinder,
+    display: 'flex',
+    flexDirection: 'column',
+    fontSize: 12,
+    fontWeight: 600,
+    span: {
+        marginBottom: 4
+    }
+}));
+
+export const StyledInput = styled.input(({ theme }) => ({
+    border: `1px solid ${theme?.colors?.mercury}`,
```

Comment:
I think we need some padding to get closer to the Zeplin mock. Here's my suggestion:
```suggestion
    paddingLeft: 12,
    border: `1px solid ${theme?.colors?.mercury}`,
```

Reply:
Also, @JonTheBurley and @michaelpuzon: the error prop can probably be a separate PR, but we will need it when implementing throughout the rest of our application when replacing existing text boxes in forms with validation.

Reply:
Sounds good, I'll put in the padding soon.